server/src/main/java/org/apache/iotdb/db/qp/physical/crud/InsertMultiTabletPlan.java

```diff
@@ -280,6 +295,21 @@ public void serializeImpl(ByteBuffer buffer) {
     }
   }
 
+  @Override
+  public void serializeToWAL(IWALByteBufferView buffer) {
+    int type = PhysicalPlanType.MULTI_BATCH_INSERT.ordinal();
+    buffer.put((byte) type);
+    buffer.putInt(insertTabletPlanList.size());
+    for (InsertTabletPlan insertTabletPlan : insertTabletPlanList) {
+      insertTabletPlan.subSerialize(buffer);
+    }
+
+    buffer.putInt(parentInsertTabletPlanIndexList.size());
+    for (Integer index : parentInsertTabletPlanIndexList) {
+      buffer.putInt(index);
+    }
+  }
+
```

Comment:
Any differences with `serializeImpl`? Add some Javadoc?

Reply:
The code logic are same, but IWALByteBufferView encapsulates some actions to deal with BufferOverflowException.

Reply:
It'll be better to add some javadoc about it.

Reply:
Is current javadoc in the IWALByteBufferView interface ok?